.travis.yml

```diff
@@ -22,7 +22,7 @@ after_success:
 	echo "Travis tag: $TRAVIS_TAG"
```

Comment:
Was space deleted here?

Reply:
No, it doesn't appear in the log and `git diff` as well

```
diff --git a/.travis.yml b/.travis.yml
index e4d7508..06d0268 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -22,7 +22,7 @@ after_success:
        echo "Travis tag: $TRAVIS_TAG"
 
         docker tag $DOCKER_NS/faas-swarm:latest-dev $DOCKER_NS/faas-swarm:$TRAVIS_TAG;
-        docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD;
+        echo $DOCKER_PASSWORD | docker login -u=$DOCKER_USERNAME --password-stdin;
         docker push $DOCKER_NS/faas-swarm:$TRAVIS_TAG;
 
         docker tag $DOCKER_NS/faas-swarm:latest-dev quay.io/$DOCKER_NS/faas-swarm:$TRAVIS_TAG;
```

Reply:
It’s showing up as red / deleted in the UI. I’ll look in another browser

Reply:
Perhaps the spacing is inconsistent on that line? Can you check?

Reply:
This appears in the master branch:
https://github.com/openfaas/faas-swarm/blob/master/.travis.yml#L22

The spaces are now updated.